server/controller/flaggedController.js

```diff
@@ -0,0 +1,46 @@
+const pool = require('../models/db');
+const queries = require('../queries/flagQuery');
+const moment = require('moment');
+
+
+
+class flagController {
+
+static async flagArticles(req,res){
```

Comment:
add this inside the article controller. you don't need this controller

Reply:
Ok sir